app/services/ipam/dhcp.rb

```diff
@@ -8,6 +8,10 @@ def suggest_ip
       ip = dhcp_proxy.unused_ip(subnet, mac)["ip"]
       logger.debug("Found #{ip}")
       ip
+    rescue => e
+      logger.warn "Failed to fetch a free IP from our proxy: #{e}"
```

Comment:
Can you also add the proxy name and URL in here? It's a bit of a pain to have to click on subnet and check what proxy is associated with the subnet